yunwoo/baekjoon/DFS-BFS/2606-바이러스.js

```diff
@@ -0,0 +1,45 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const node = Number(input[0]);
+  const edges = input.slice(2).map((el) => el.split(" ").map(Number));
+  const graph = Array.from({ length: node + 1 }, () => []);
+  const visited = Array.from({ length: node + 1 }, () => false);
```

Comment:
```suggestion
  const visited = Array(node + 1).fill(false);
```
- 채우는 값이 배열이 아닌 원시값인 false이니 위와 같이 쓰는 방법도 있씁니다 좀 더 깔끔하네요 :)

Reply:
- 헉 그렇네요 감사합니다 👍